source/_integrations/water_heater.markdown

```diff
@@ -3,6 +3,7 @@ title: Water Heater
 description: Instructions on how to setup water heater devices within Home Assistant.
 ha_release: 0.81
 ha_domain: water_heater
+ha_iot_class: Calculated
```

Comment:
This is an entity component, used for other integration to build on.

It cannot be used by itself, hence it doesn't have an IoT class.

```suggestion
ha_iot_class: ~
```